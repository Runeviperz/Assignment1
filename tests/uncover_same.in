g 10 10
b 0 0
b 0 2
b 3 5
b 5 9
b 3 4
b 2 9
b 2 3
b 1 3
b 7 4
b 7 5
u 1 1
u 1 5
u 1 1
