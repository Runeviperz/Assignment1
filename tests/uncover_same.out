g 10 10
b 0 0
b 0 2
b 3 5
b 5 9
b 3 4
b 2 9
b 2 3
b 1 3
b 7 4
b 7 5
+----------+
|**********|
|**********|
|**********|
|**********|
|**********|
|**********|
|**********|
|**********|
|**********|
|**********|
+----------+
u 1 1
+----------+
|**********|
|*2********|
|**********|
|**********|
|**********|
|**********|
|**********|
|**********|
|**********|
|**********|
+----------+
u 1 5
+----------+
|**********|
|*2********|
|**********|
|**********|
|**********|
|*0********|
|**********|
|**********|
|**********|
|**********|
+----------+
error
